src/js/hca/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,92 @@
+        <p>Equal to VA Form 10-10EZ (Application for Health Benefits).</p>
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          resumeOnly
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}/>
+        <div className="process schemaform-process">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>To fill out this application, you’ll need your:</h6></div>
+              <ul>
+                <li>Social Security number (required)</li>
+                <li>Copy of your military discharge papers (DD214 or other separation documents)</li>
+                <li>Financial information—and your dependents’ financial information</li>
+                <li>Most recent tax return</li>
+                <li>Account numbers for any health insurance you currently have (such as Medicare, private insurance, or insurance from an employer)</li>
+              </ul>
+              <p><strong>What if I need help filling out my application?</strong> An accredited representative with a Veterans Service Organization (VSO) can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Find an accredited representative</a>.</p>
+              <div><h6>Vision and dental benefits</h6></div>
+              <p>You may qualify for vision and dental benefits as part of your VA health care benefits.</p>
+              <ul>
+                <li><strong>Vision benefits.</strong> VA health care covers routine eye exams and preventive tests. In some cases, you may get coverage for eyeglasses or services for blind or low-vision rehabilitation. <a href="/health-care/about-va-health-care/vision-care/">Learn more about vision care through VA</a>.</li>
+                <li><strong>Dental benefits.</strong> In some cases, you may receive dental care as part of your health care benefits. <a href="/health-care/about-va-health-care/dental-care/">Learn more about VA dental services</a>.</li>
+              </ul>
+              <div><h6>Health care priority groups</h6></div>
+              <p>When you apply for VA health care, we’ll assign you 1 of 8 priority groups. This system helps to make sure that Veterans who need immediate care can get signed up quickly. We assign Veterans with service-connected disabilities the highest priority. We assign the lowest priority to Veterans who earn a higher income and who don’t have any service-connected disabilities. <a href="/health-care/eligibility/#priority-groups">Learn more about priority groups</a>.</p>
+            </li>
+            <li className="process-step list-two">
+              <div><h5>Apply</h5></div>
+              <p>Complete this health care benefits form.</p>
+              <p>After submitting the form, you’ll get a confirmation message. You can print this for your records.</p>
```

Comment:
@peggygannon @stacyhlinn Would it be of interest to provide a similar level of specificity regarding what details will be provided on the confirmation page as is used to list the required information above (e.g. 

You will receive: 
- a confirmation number
- a mailing address to send any supporting documents)

Reply:
I think this is a good idea if we can still make it concise. It's a fine balance between providing enough info to apply vs too much information that cause cognitive burden.

Reply:
@cehsu Yes, the more specific we can be, the better. Is it correct that the confirmation message only contains the confirmation number and a mailing address for any supporting doc? Does it list anything else?
And is this true for all forms? If we decide to update this language here, I want to be sure we update it on the other form introduction pages. 

@stacyhlinn Let me know your thoughts on this....

Reply:
@stacyhlinn Yes, agree. I wouldn't want to add a long list of things. @cehsu can you confirm that it's just a confirmation number and address that's provided?

Reply:
@peggygannon @cehsu HCA actually does not show either of those things, just a timestamp. Other forms show different things, but generally they have at least a confirmation number and an address.

Reply:
thanks, @jbalboni 
@cehsu I think it's best if we leave the sentence as it is currently written. Since each form shows different things, it's best to keep this message general.